test/extended/util/annotate/rules.go

```diff
@@ -159,6 +163,7 @@ var (
 			`\[sig-network\]\[Feature:Router\] The HAProxy router should serve the correct routes when scoped to a single namespace and label set`,
 			`\[sig-network\]\[Feature:Router\] The HAProxy router should set Forwarded headers appropriately`,
 			`\[sig-network\]\[Feature:Router\] The HAProxy router should support reencrypt to services backed by a serving certificate automatically`,
+			`\[sig-network\] Networking should provide Internet connection for containers`,
```

Comment:
This belongs above. These tests are broken into two sections, ones that always require internet access and ones that are broken because of the ICSP bug.